src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -4,13 +4,14 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
+    <PreReleaseLabel>servicing</PreReleaseLabel>
```

Comment:
This package isn't the one that contains sni.dll itself, that is in the individual RID packages, this package just references all the RID split SNI packages.